src/Tmds.Ssh/SshAgent.cs

```diff
@@ -0,0 +1,282 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Sockets;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Tmds.Ssh
+{
+    // https://datatracker.ietf.org/doc/html/draft-miller-ssh-agent - SSH Agent Protocol
+    sealed class SshAgent : IDisposable
+    {
+        private const MessageId SSH_AGENTC_REQUEST_IDENTITIES = (MessageId)11;
```

Comment:
Is there a reason why you are using this as a const here rather than just adding it to the enum? I assume these ids are still unique even if it's specific to ssh-agent?

Reply:
These ids are specific to SSH agent and overlap in value with the the regular SSH message ids.